content/docs/authorization/cli.md

```diff
@@ -1095,3 +1096,42 @@ karavictl tenant delete [flags]
 $ karavictl tenant delete --name Alice
 ```
 On success, there will be no output. You may run `karavictl tenant get --name <tenant-name>` to confirm the deletion occurred.
+
+
+
+---
+
+
+
+### karavictl tenant update
+
+Updates a tenant's resource within CSM
+
+##### Synopsis
+
+Updates a tenant resource within CSM (currently only for powerflex)
```

Comment:
no need to mention storage system applicability here

Reply:
Removed reference to PowerFlex here